spirv_hlsl.cpp

```diff
@@ -4850,6 +4867,8 @@ string CompilerHLSL::compile()
 	validate_shader_model();
 	update_active_builtins();
 	analyze_image_and_sampler_usage();
+	if (get_execution_model() == ExecutionModelFragment)
+		analyze_interlocked_resource_usage();
```

Comment:
Should only be used if ROV execution modes are active. No need to analyze here unless we need to.